hazelcast/src/main/java/com/hazelcast/internal/memory/MemoryStats.java

```diff
@@ -35,7 +47,7 @@
      *
      * @return total physical memory in bytes.
      */
-    @Probe(level = MANDATORY)
+    @Probe(name = MEMORY_METRIC_TOTAL_PHYSICAL, level = MANDATORY)
```

Comment:
How about specifying `BYTES` unit here and below?

Reply:
Added `BYTES` everywhere.